packages/workbox-core/src/types.ts

```diff
@@ -66,6 +77,18 @@ export interface RouteHandlerCallback {
   (options: RouteHandlerCallbackOptions): Promise<Response>;
 }
 
+/**
+ * The "handler" callback is invoked whenever a `Router` matches a URL/Request
+ * to a `Route` via its `RouteMatchCallback`. This handler callback should
+ * return a `Promise` that resolves with a `Response`.
+ *
+ * If a non-empty array or object is returned by the `RouteMatchCallback` it
+ * will be passed in as this handler's `options.params` argument.
+ */
+export interface HandlerCallback {
```

Comment:
This looks identical to the `RouteHandlerCallback` type above, and it doesn't seem to be used anywhere in the code base. Is it needed?

Reply:
So my intention here was to export an interface that developers could implement when they wrote their own "manual" handler functions. But `HandlerCallback `, as implemented, is probably too generic, since it could take either `RouteHandlerCallbackOptions` or `ManualHandlerCallbackOptions` as its parameter.

I've gotten more specific by removing `HandlerCallback` and exporting an explicit `ManualHandlerCallback` instead, that takes `ManualHandlerCallbackOptions`.

So even though `ManualHandlerCallbackOptions` isn't used by the codebase, developers could take advantage of it via

```
import {ManualHandlerCallback} from 'workbox-core/types';

const myHandler: ManualHandlerCallback = async({request, event}) => {
  const networkFirst = new NetworkFirst();
  const networkResponse = networkFirst.handle({event, request});
  return networkResponse || new Response('...');
};
```

Reply:
SGTM